LEAF_Nexus/sources/Config.php

```diff
@@ -26,8 +26,6 @@ public function __construct(array $site_paths, array $oc_settings)
     {
         $this->title = $oc_settings['heading'];
         $this->city = $oc_settings['subheading'];
-        $this->adPath = $oc_settings['adPath'];
         self::$uploadDir = $site_paths['site_uploads'];

```

Comment:
Yes, I think the adPath is unused at a local site level now, but need someone to double check the configs in prod.

If it turns out that prod needs it, we might need to update the dev environment to avoid the warnings.

Added this as a prerequisite item.